src/pages/index.js

```diff
@@ -69,53 +69,153 @@ const moreLinks = [
-        alt=""
-        style={{ marginBottom: `var(--space-3)` }}
-      />
-      <h1>
-        Welcome to <b>Gatsby!</b>
-      </h1>
-      <p className={styles.intro}>
-        <b>Example pages:</b>{" "}
-        {samplePageLinks.map((link, i) => (
-          <React.Fragment key={link.url}>
-            <Link to={link.url}>{link.text}</Link>
-            {i !== samplePageLinks.length - 1 && <> · </>}
-          </React.Fragment>
-        ))}
-        <br />
-        Edit <code>src/pages/index.js</code> to update this page.
-      </p>
-    </div>
-    <ul className={styles.list}>
-      {links.map(link => (
-        <li key={link.url} className={styles.listItem}>
-          <a
-            className={styles.listItemLink}
-            href={`${link.url}${utmParameters}`}
+    {/* <h1 className="text-3xl font-bold underline">Hello world!</h1> */}
+    {/* <label htmlFor="firstName">First Name:</label>
+    <input type="text" id="firstName" className="inputFormat" />
+    <br></br>
+    <label htmlFor="lastName">Last Name:</label>
+    <input type="text" id="lasttName" className="inputFormat" />
+    <br></br>
+    <label htmlFor="age">Age:</label>
+    <input type="number" id="age" className="inputFormat w-20" />
+    <br></br>
+    <label htmlFor="years">Years:</label>
+    <input type="number" id="years" className="inputFormat w-20" /> */}
+    <div className="flex flex-col justify-between bg-yellow-300 sm:bg-green-400 border rounded py-2 px-2">
+      {/* Think of first one (default) as the mobile, which can be override with larger ones */}
+      <div className="dataFormat">
+        <div className="w-24">
+          <label className="labelChangeColor" htmlFor="firstName">
+            First Name:
+          </label>
+        </div>
+        <div className="left-5">
+          <input type="text" id="firstName" className="inputFormat" />
+        </div>
+      </div>
+      <div className="dataFormat">
+        <div className="w-24">
+          <label className="labelChangeColor" htmlFor="lastName">
+            Last Name:
+          </label>
+        </div>
+        <div>
+          <input type="text" id="lastName" className="inputFormat" />
+        </div>
+      </div>
+      <div className="dataFormat">
+        <div className="w-24">
+          <label className="labelChangeColor" htmlFor="age">
+            Age:
+          </label>
+        </div>
+        <div>
+          <input type="number" id="age" className="inputFormat w-20" />
+        </div>
+      </div>
+      <div className="dataFormat">
+        <div className="w-24">
+          <label className="labelChangeColor" htmlFor="years">
+            Years:
+          </label>
+        </div>
+        <div>
+          <input type="number" id="years" className="inputFormat w-20" />
+        </div>
+      </div>
+      <div className="flex h-4"></div>
+      <div>
+        <div className="flex content-center">
+          {/* Don't really use this type anymore */}
```

Comment:
Delete any commented out files.

Reply:
Another I'll keep for my notes, but realize would not go into production.